### DISCUSSION TITLE
Welcome to 🦏 discussions!

### DISCUSSION THREAD

**Discussion by marekrogala** [Appsilon/rhino#278] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other users of Rhino and members of our community. 

We would love to hear your feedback, and we welcome contributions and ideas.

We hope that you use Dicussions to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Help other community members with using Rhino.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bharath-cchmc** [comment]
Hi. I am Bharath from the US. I'm testing / MVP building a SaaS app with RShiny (not a typical use case for using Rshiny). I am seeing if I can build a login page and then a dashboard.
So how to pass reactive values between box modules?
